src/pages/trade/[pair].tsx

```diff
@@ -43,8 +43,16 @@ export default function Page() {
   const router = useRouter();
   const { pair } = router.query;
 
-  let token: ReturnType<typeof getToken> = Token.SOL;
-  let currency: ReturnType<typeof getComparisonCurrency> = "usd";
+  if (!pair) {
```

Comment:
why is this necessary? why can't we just default to a pair in the case where one isn't provided in the url?

Reply:
yes the default is now defaulting in the index page